backend-shared/config/db/clustercredentials_test.go

```diff
@@ -0,0 +1,69 @@
+package db
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateandDeleteClusterCredentials(t *testing.T) {
+	ctx := context.Background()
+	dbq, err := NewUnsafePostgresDBQueries(true, true)
+	if !assert.NoError(t, err) {
+		return
+	}
+	defer dbq.CloseDatabase()
+	clusterCreds := ClusterCredentials{
+		Host:                        "test-host",
+		Kube_config:                 "test-kube_config",
+		Kube_config_context:         "test-kube_config_context",
+		Serviceaccount_bearer_token: "test-serviceaccount_bearer_token",
+		Serviceaccount_ns:           "test-serviceaccount_ns",
+	}
+	err = dbq.CreateClusterCredentials(ctx, &clusterCreds)
+	if !assert.NoError(t, err) {
+		return
+	}
```

Comment:
In order to ensure the create was successful, we can add a Get after, and verify the diff.